### PR TITLE
ADD: improve russian locale

### PR DESCRIPTION
**Pull request by WarmongeR1**

### CODE REVIEW

**Comment by bear** [comment]
@WarmongeR1 can you check on that failing test that Travis is showing?

**Comment by WarmongeR1** [comment]
Fixed

File: parsedatetime/pdt_locales/en_A.py [hunk 1]
```diff
@@ -0,0 +1,27 @@
+# -*- encoding: utf-8 -*-
```
Comment (philiptzou) [inline parsedatetime/pdt_locales/en_A.py]: Shouldn't this file be "en_AU.py" to follow naming convention?

File: parsedatetime/tests/TestRussianLocale.py [hunk 2]
```diff
@@ -78,6 +78,24 @@ def testDates(self):
         self.assertExpectedResult(self.cal.parse('25.8', start), (target, 1))
         self.assertExpectedResult(self.cal.parse('25.08', start), (target, 1))
 
+    def testDatesLang(self):
+        if sys.version_info >= (3, 0):
+            target = datetime.datetime(2006, 8, 25, 23, 5).timetuple()
```
Comment (philiptzou) [inline parsedatetime/tests/TestRussianLocale.py]: Well this's strange, test only applies to PY3K?

**Comment by philiptzou** [comment]
Hold on... Still some problems. Please check my comment @WarmongeR1

**Comment by bear** [comment]
@philiptzou sorry - I eager-clicked the merge button!

I've fixed the spelling of the en_AU.py file.

File: parsedatetime/pdt_locales/en_A.py (hunk 1 above)
Comment (WarmongeR1) [inline parsedatetime/pdt_locales/en_A.py]: I got name files from ```self.localeID``` variable.
I think for ```pdtLocale_au``` this variable should be equal ```en_AU```, but compatibility....

File: parsedatetime/tests/TestRussianLocale.py (hunk 2 above)
Comment (WarmongeR1) [inline parsedatetime/tests/TestRussianLocale.py]: Yes. PY2K have error in this code - ```self.cal.parse``` returns current time (```datetime.datetime.now()```)

File: parsedatetime/pdt_locales/en_A.py (hunk 1 above)
Comment (bear) [inline parsedatetime/pdt_locales/en_A.py]: ah - that's a mistake then - the localeID should have been set to en_AU 

we will have to fix that and then bump the major version number IMO

File: parsedatetime/tests/TestRussianLocale.py (hunk 2 above)
Comment (philiptzou) [inline parsedatetime/tests/TestRussianLocale.py]: Yeah but it shouldn't be inconsistent between PY2x and PY3K. We'd better figure it out why it fails when using Python 2.x.

Comment (bear) [inline parsedatetime/tests/TestRussianLocale.py]: @WarmongeR1 can you file an issue about the PY2k error and reference this conversation so we can know it needs fixing

File: parsedatetime/pdt_locales/en_A.py (hunk 1 above)
Comment (philiptzou) [inline parsedatetime/pdt_locales/en_A.py]: Agreed. Bump it.

**Pull request merged**